algorithms/strings/longest_palindromic_substring.py

```diff
@@ -0,0 +1,44 @@
+'''
+Given string s, find the longest palindromic substring.
+
+Example1:
+
+* input: "babad"
+* output: "bab" or "aba"
+
+Example2:
+
+* input: "cbbd"
+* output: "bb"
+
+Manacher's algorithm
+
+'''
+
+def longestPalindrome(s):
```

Comment:
Convert to snake case (```longest_palindrome```), please.

Reply:
Ok, thanks for your suggestion, and I have already fixed the case with using much more complex testing case.

Reply:
I was suggesting to change the name of the function from came case (```longestPalindrome```) to snake case (```longest_palindrome```), to match the format from the other files.

Reply:
ooo, lol, get it.